File: PHP/KODEPROGRAM/Baju.php
<?php
require_once "Aksesoris.php";

class Baju extends Aksesoris {
    public $untuk;
    public $size;
    public $merk;

    public function __construct($id, $nama_produk, $harga_produk, $stok_produk, $foto_produk, $jenis, $bahan, $warna, $untuk, $size, $merk) {
        parent::__construct($id, $nama_produk, $harga_produk, $stok_produk, $foto_produk, $jenis, $bahan, $warna);
        $this->untuk = $untuk;
        $this->size = $size;
        $this->merk = $merk;
    }

    public function getUntuk() {
        return $this->untuk;
    }

    public function setUntuk($untuk) {
        $this->untuk = $untuk;
    }

    public function getSize() {
        return $this->size;
    }

    public function setSize($size) {
        $this->size = $size;
    }

    public function getMerk() {
        return $this->merk;
    }

    public function setMerk($merk) {
        $this->merk = $merk;
    }
}
?>

File: CPP/KODEPROGRAM/Baju.cpp
#include <bits/stdc++.h>
#include "Aksesoris.cpp"

using namespace std;

class Baju : public Aksesoris {
	private:
		string untuk;
		string size;
		string merk;

	public:
		Baju(int id, string nama_produk, int harga_produk, int stok_produk,
		string jenis, string bahan, string warna, string untuk, string size, string merk)
		: Aksesoris(id, nama_produk, harga_produk, stok_produk, jenis, bahan, warna) {
			this->untuk = untuk;
			this->size = size;
			this->merk = merk;
		}
		
		void setUntuk(string untuk){
			this->untuk = untuk;
		}
		
		string getUntuk(){
			return untuk;
		}

		void setSize(string size){
			this->size = size;
		}
		
		string getSize(){
			return size;
		}

		void setMerk(string merk){
			this->merk = merk;
		}
		
		string getMerk(){
			return merk;
		}
};